front/app/modules/commercial/moderation/admin/containers/ModerationRow.tsx

```diff
@@ -181,77 +194,91 @@ const ModerationRow = memo<Props & InjectedIntlProps>(
           {moment(moderation.attributes.created_at).format('L')}{' '}
           {moment(moderation.attributes.created_at).format('LT')}
         </td>
-        <td className="type">{contentType}</td>
+        <td className="type">
+          {formatMessage(
+            {
+              Idea: messages.post,
+              Comment: messages.comment,
+              Initiative: messages.initiative,
+            }[moderatableType]
+          )}
+        </td>
         <td className="belongsTo">
-          {Object.keys(moderation.attributes.belongs_to).length > 0 &&
-            Object.keys(moderation.attributes.belongs_to).map((key, index) => (
-              <BelongsToItem
-                key={`${moderation.id}-${key}`}
-                className={
-                  index + 1 ===
-                  Object.keys(moderation.attributes.belongs_to).length
-                    ? 'last'
-                    : ''
-                }
-              >
-                <BelongsToType>
-                  <FormattedMessage
-                    {...({
-                      idea: messages.post,
-                      project: messages.project,
-                      initiative: messages.initiative,
-                    }[key] as any)}
-                  />
-                  :
-                </BelongsToType>
-                <a
-                  href={`/${key === 'idea' ? 'idea' : 'project'}s/${
-                    moderation.attributes.belongs_to[key].slug
-                  }`}
-                  role="button"
-                  onClick={handleBelongsToLinkOnClick}
-                  data-belongstotype={key}
-                >
-                  <T
-                    value={moderation.attributes.belongs_to[key].title_multiloc}
-                  />
-                </a>
-              </BelongsToItem>
-            ))}
+          {belongsToTypes.length > 0 ? (
+            belongsToTypes.map((belongsToType: TBelongsTo, index) => {
+              const belongsToTypeMessage = {
+                idea: messages.post,
+                project: messages.project,
+                initiative: messages.initiative,
+              }[belongsToType];
```

Comment:
Because this is how the belongs_to part is returned by the backend. The others are moderatable_type, which are for some reason capitalized. Trying to keep the code simple (and not doing .toLowercase() or something) this way.